### ISSUE TITLE
Google play rejected app for ACTIVITY_RECOGNITION

### ISSUE THREAD

**Issue by vladretca** [transistorsoft/cordova-background-geolocation-lt#1455]
## Your Environment
* Plugin version: 4.16.5
* Platform: Android
* OS version: Android 14
* Device manufacturer / model: All
* Cordova version (`cordova -v`): 12.0.0
* Cordova platform version (`cordova platform ls`): android 13.0.0
* Plugin config provided to `#ready`:
```javascript <!-- syntax-highlighting:  paste your code below -->
{
        reset: true,
        debug: environment.environment != 'production',
        logLevel: BackgroundGeolocation.LOG_LEVEL_VERBOSE,
        desiredAccuracy: BackgroundGeolocation.DESIRED_ACCURACY_HIGH,
        distanceFilter: 70,
        url: '/url',
        stopOnTerminate: true,
        startOnBoot: false,
        autoSync: true,
        autoSyncThreshold: 7,
        batchSync: true,
        preventSuspend: true,
        // heartbeatInterval: 60,
        triggerActivities: 'in_vehicle',
        foregroundService: false,
        activityType: BackgroundGeolocation.ACTIVITY_TYPE_AUTOMOTIVE_NAVIGATION,
        motionTriggerDelay: 30000,
        maxBatchSize: 50,
        notification: {
          title: this.translate.instant('service.location-trackers.subheader')
        },
        authorization: {
          strategy: 'JWT',
          accessToken: token,
        },
        locationAuthorizationRequest: 'Always',
        backgroundPermissionRationale: {
          title: this.translate.instant('service.location-trackers.backgroundPermission.title'),
          message: this.translate.instant('service.location-trackers.backgroundPermission.message'),
          positiveAction: this.translate.instant('service.location-trackers.backgroundPermission.positiveAction'),
          negativeAction: this.translate.instant('generic.cancel')
        }
      }
```

## Expected Behavior
Pass google play app content 

## Actual Behavior
I received a email from google that requires a app content policy for "Health app" because of "android.permission.ACTIVITY_RECOGNITION" permission, and unlisted my release. 
I know the plugin uses this permission to preserve battery life, but it's not a Health app and none their options apply to me.

What solution do I have, to pass this, but also to have background geolocation, and geofencing?

## Steps to Reproduce
N/A

## Context
N/A

## Debug logs
N/A

**Comment by christocracy** [comment]
See my blog:  https://transistorsoft.medium.com/google-play-store-required-declarations-for-foreground-services-and-health-97da6e2abd5a

**Comment by vladretca** [comment]
I tried it, doesn't work. I got rejected again

**Comment by haziqali-2019** [comment]
same issue..i checked the blog and added the permission declaration for activity (which is in review) but i can't find anywhere in the declaration to update the declaration for foreground service

**Comment by Talentpro** [comment]
same issue here, added ACTIVITY_RECOGNITION declaration in policy and app updates were accepted for around two months. Now app update got rejected with same issue.

Health Connect by Android Permissions policy: Inappropriate Health Connect Access Requested.

**Comment by christocracy** [comment]
If you cannot convince Google Play to authorize the use of the motion api, then you have no other option but to remove that permission from your AndroidManifest, as documented in my blog, and deal with the consequences of reduced performance of the plug-in.

**Comment by chrisbiscuit** [comment]
Hi all,

I wanted to share a helpful post related to a Google Play rejection issue I faced, which could assist others dealing with similar challenges. After struggling with repeated rejections over the Health Declaration form, I was able to resolve it by removing the **"Activity & Fitness**" category and clarifying that our app doesn’t provide any health features.

I’ve detailed the steps I took in the following GitHub post, which may be useful for anyone facing similar issues with Google Play’s policies:

[How we resolved Google Play rejection](https://github.com/transistorsoft/capacitor-background-geolocation/issues/289)

Hope it helps!
